pandas/io/packers.py

```diff
@@ -558,19 +564,23 @@ def create_block(b):
 
 
 def pack(o, default=encode,
-         encoding='latin1', unicode_errors='strict', use_single_float=False):
+         encoding='latin1', unicode_errors='strict', use_single_float=False,
+         autoreset=1, use_bin_type=1):
```

Comment:
`use_bin_type` is for a [new binary format](https://github.com/msgpack/msgpack/blob/master/spec.md#formats-bin); [str 8](https://github.com/msgpack/msgpack/blob/master/spec.md#formats-str) was introduced at the same time, so that arg enables both, though they needn't be linked.  I think it's good to differentiate binary and strings, since in python there is a distinction.  In PY2 you wouldn't be able to round-trip `str`--which is really bytes--using utf 16 or utf 32 encoding without that enabled.  `autoreset` is not new.  I don't know if the omission from `packers` was intentional or not.